### PR TITLE
Random Event Tweaks

### PR DESCRIPTION
**Pull request by Neerti**
Tweaks event text to fit with the lore better.
Adds more informative text for vending machine random event.
Removes RNG viruses.
Removes all references to the Icarus that players would normally see.

### CODE REVIEW

File: code/modules/events/brand_intelligence.dm [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 
 /datum/event/brand_intelligence/announce()
-	command_announcement.Announce("Rampant brand intelligence has been detected aboard [station_name()], please stand-by.", "Machine Learning Alert")
+	command_announcement.Announce("An ongoing mass upload of malware for vendors has been detected onboard  [station_name()], which appears to transmit \
+	to other nearby vendors.  The original infected machine is believed to be a [originMachine.name].", "Vendor Service Alert")
```
Comment (PsiOmegaDelta) [inline code/modules/events/brand_intelligence.dm]: \a [originMachine]

File: code/controllers/shuttle_controller.dm [hunk 2]
```diff
@@ -286,9 +286,9 @@ var/global/datum/shuttle_controller/shuttle_controller
 		"Mining Station" = locate(/area/skipjack_station/mining)
 		)
 
-	VS.announcer = "NDV Icarus"
-	VS.arrival_message = "Attention, [station_short], we just tracked a small target bypassing our defensive perimeter. Can't fire on it without hitting the station - you've got incoming visitors, like it or not."
-	VS.departure_message = "Your guests are pulling away, [station_short] - moving too fast for us to draw a bead on them. Looks like they're heading out of the system at a rapid clip."
+	VS.announcer = "Colony Automated Radar Array"
+	VS.arrival_message = "Attention.  Unidentified object with no signature detected approaching the colony."
```
Comment (Zuhayr) [inline code/controllers/shuttle_controller.dm]: Wouldn't 'no signature' make it undetectable?

Comment (Neerti) [inline code/controllers/shuttle_controller.dm]: I probably used the wrong word there, it was meant to be more of 'the object is not telling us what it is'.

File: code/modules/events/rogue_drones.dm [hunk 3]
```diff
@@ -44,6 +53,6 @@
 		num_recovered++
 
 	if(num_recovered > drones_list.len * 0.75)
-		command_announcement.Announce("Icarus drone control reports the malfunctioning wing has been recovered safely.", "Rogue drone alert")
+		command_announcement.Announce("The drones that were malfunctioning has been recovered safely.", "Rogue drone alert")
```
Comment (Zuhayr) [inline code/modules/events/rogue_drones.dm]: *have

File: code/controllers/shuttle_controller.dm (hunk 2 above)
Comment (Zuhayr) [inline code/controllers/shuttle_controller.dm]: Feels like 'unknown signature' or just 'unidentified object approaching' would read better.

File: code/modules/events/gravity.dm [hunk 4]
```diff
@@ -5,7 +5,8 @@
 	endWhen = rand(15, 60)
 
 /datum/event/gravity/announce()
-	command_announcement.Announce("Feedback surge detected in mass-distributions systems. Artificial gravity has been disabled whilst the system reinitializes. Further failures may result in a gravitational collapse and formation of blackholes.", "Gravity Failure")
+	command_announcement.Announce("Feedback surge detected in mass-distributions systems. Artificial gravity has been disabled whilst the system \
```
Comment (Zuhayr) [inline code/modules/events/gravity.dm]: Should this be 'mass-distribution systems'?

File: code/modules/events/infestation.dm [hunk 5]
```diff
@@ -23,7 +22,7 @@
 
 /datum/event/infestation/start()
 
-	location = rand(0,9)
+	location = rand(0,8)
```
Comment (Zuhayr) [inline code/modules/events/infestation.dm]: Not a priority but this should really be something like rand(LOC_KITCHEN,LOC_TACTICAL) or something.

**Pull request merged**